Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -169,27 +168,19 @@ final class PledgeCTAContainerView: UIView {
 
 private func adaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
   return { (stackView: UIStackView) in
-    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
     let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : 0)
 
     return stackView
-      |> \.axis .~ axis
+      |> \.axis .~ NSLayoutConstraint.Axis.horizontal
       |> \.spacing .~ spacing
   }
 }
 
-private func pledgeCTAButtonStyle(
-  _ isAccessibilityCategory: Bool, amountAndRewardTitleStackViewIsHidden: Bool
-) -> (ButtonStyle) {
-  return { (button: UIButton) in
-    let lineBreakMode: NSLineBreakMode = isAccessibilityCategory || amountAndRewardTitleStackViewIsHidden
-      ? NSLineBreakMode.byWordWrapping : NSLineBreakMode.byTruncatingTail
-
-    return button
-      |> roundedStyle(cornerRadius: 12)
-      |> UIButton.lens.titleLabel.font .~ UIFont.ksr_headline(size: 15)
-      |> UIButton.lens.layer.borderWidth .~ 0
-      |> (UIButton.lens.titleLabel .. UILabel.lens.textAlignment) .~ NSTextAlignment.center
-      |> (UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode) .~ lineBreakMode
-  }
+private let pledgeCTAButtonStyle: ButtonStyle = { button in
```

Comment:
It shouldn't be necessary to set `roundedStyle` and `borderWidth` because these are included in our base button styles now. If we want all of our buttons to behave this way shouldn't we move these to the base style?